bin/setup

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
I always found this script kind of useless, what do you think? I tend to remove it.